crates/router/src/routes/app.rs

```diff
@@ -759,22 +759,14 @@ impl Routing {
                 },
             )))
             .service(
-                web::resource("/default")
-                    .route(web::get().to(|state, req| {
-                        routing::routing_retrieve_default_config(
-                            state,
-                            req,
-                            &TransactionType::Payment,
-                        )
-                    }))
-                    .route(web::post().to(|state, req, payload| {
-                        routing::routing_update_default_config(
-                            state,
-                            req,
-                            payload,
-                            &TransactionType::Payment,
-                        )
-                    })),
+                web::resource("/default").route(web::post().to(|state, req, payload| {
+                    routing::routing_update_default_config(
```

Comment:
Can we check if this is also not being used then maybe we can remove this

Reply:
@ThisIsMani are we using this route in Dashboard as internally there is no dependency on this ?

Reply:
Please check with @JeevaRamu0104 once.

Reply:
This is checked out with @JeevaRamu0104 , we don't have these endpoints being used in dashboard.